test/integration/error_spam_test.go

```diff
@@ -118,6 +118,12 @@ func TestErrorSpam(t *testing.T) {
 		}
 	}
 
+	logsToBeRemoved, err := filepath.Glob(filepath.Join(logDir, "minikube_*"))
+	if err != nil {
+		t.Error("failed to clean old log files", err)
+	}
+	cleanupLogFiles(t, logsToBeRemoved)
```

Comment:
I don't see the need for removing all minikube logs when we delete the logs for each specific command below (line 170).